src/android/camera_hal_config.cpp

```diff
@@ -357,15 +357,8 @@ CameraHalConfig::CameraHalConfig()
  */
 int CameraHalConfig::parseConfigurationFile()
 {
-	std::filesystem::path filePath = LIBCAMERA_SYSCONF_DIR;
-	filePath /= "camera_hal.yaml";
-	if (!std::filesystem::is_regular_file(filePath)) {
-		LOG(HALConfig, Debug)
-			<< "Configuration file: \"" << filePath << "\" not found";
-		return -ENOENT;
-	}
-
-	FILE *fh = fopen(filePath.c_str(), "r");
+	const char *filePath = LIBCAMERA_SYSCONF_DIR "camera_hal.yaml";
```

Comment:
Does this correctly join the two paths? What happens if LIBCAMERA_SYSCONF_DIR doesn't have a / at the end?

Reply:
Nice catch. Fixed.